internal/pkg/manifest/workload.go

```diff
@@ -38,12 +40,13 @@ var (
 	subnetPlacements = []string{PublicSubnetPlacement, PrivateSubnetPlacement}
 
 	// Error definitions.
-	errUnmarshalBuildOpts  = errors.New("cannot unmarshal build field into string or compose-style map")
-	errUnmarshalCountOpts  = errors.New(`cannot unmarshal "count" field to an integer or autoscaling configuration`)
-	errUnmarshalRangeOpts  = errors.New(`cannot unmarshal "range" field`)
-	errUnmarshalExec       = errors.New("cannot unmarshal exec field into boolean or exec configuration")
-	errUnmarshalEntryPoint = errors.New("cannot unmarshal entrypoint into string or slice of strings")
-	errUnmarshalCommand    = errors.New("cannot unmarshal command into string or slice of strings")
+	errUnmarshalBuildOpts    = errors.New("cannot unmarshal build field into string or compose-style map")
```

Comment:
nit: can we remove "cannot" from the error messages?

Reply:
changed to "unable to"